src/View/Form/NullContext.php

```diff
@@ -36,9 +36,8 @@ class NullContext implements ContextInterface
      * Constructor.
      *
      * @param \Cake\Network\Request $request The request object.
-     * @param array $context Context info.
      */
-    public function __construct(Request $request, array $context)
+    public function __construct(Request $request)
```

Comment:
The problem is that the parameter may be unused because of two reasons:

* It's really not supposed to be used and should be removed.
* It's forgotten to be used and the code should be fixed. Like in this code sample:

```
private $foo;

private $bar;

public function __construct($foo, $bar)
{
    $this->foo = $foo;
}
```

This check actually found a few of these situations in our codebase so I'm convinced it should be part of the default level 0.